_includes/officer-for-sidebar.html

```diff
@@ -1,28 +1,40 @@
-{% if officer %}
+{% assign officers = site.data.regnum | where:"group", "Insulae Draconis" | where:"office", include.office | sort: 'responsibility' %}
+{% assign numofficers = officers|size %}
+{% assign role = site.data.officetitles | where:"slug", include.office | first %}
 
-{% assign role = site.data.officetitles | where:"slug", officer.office | first %}
+{% if numofficers > 0 %}
 
-<h3>{{ role.title }}</h3>
-<p><strong>{{ officer.scaname }}</strong>
-{% unless officer.mundanename == "" %}
- ({{ officer.mundanename }})
-{% endunless %}
-{% unless officer.pronouns == "" %}
- <span class="text-muted">{{ officer.pronouns }}</span>
-{% endunless %}<br>
-<strong>Contact:</strong> {{ officer.email }}
-</p>
+  <h3>{{ role.title }}</h3>
 
-{% if role.report %}<p><a href="{{ role.report }}">How to report</a></p>{% endif %}
+  {% for officer in officers %}
 
-{% if role.desc %}
-<p>
-{{ role.desc }}
-</p>
-{% endif %}
+    {% unless officer.responsibility == " " %}
```

Comment:
Q: this has a space, not the empty string, is that intentional?

Reply:
It is; the regnum seems to insert a space for a primary officer.

Reply:
Came to think of 2 things:

- Would liquid have a `strip` function?
- Would liquid have an `empty?/blank?` function?

Reply:
In addition, I learned about the `== empty` check! https://shopify.github.io/liquid/basics/types/#emptydrop